scripts/v.db.addcolumn/v.db.addcolumn.html

```diff
@@ -46,8 +46,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 Moritz Lennert (mlennert@club.worldonline.be)
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 41 

`<a href="v.db.update.html">v.db.update</a><br>` 

should be 

`<a href="v.db.update.html">v.db.update</a>,`.

Reply:
I don't think so: not a module, but an overview page is following.
I have changed it to an empty line in between like elsewhere.